devops/dockerfiles/origin-relayer

```diff
@@ -0,0 +1,22 @@
+FROM node:10 as build
+
+WORKDIR /app
+
+ENV NODE_ENV=production
+
+# Install envkey-source to make environment available for sequelize migration
```

Comment:
We can remove this since there is no sequelize-migration and the relayer has the envkey package

Reply:
Before the launch, the relayer service is going to need to have a few DB tables to record transactions and keep track of various signals for fraud purposes.
So as opposed to removing it, then add it in a few days from now, ok if we keep as is ?

Reply:
Yep sure, wasn't sure if that was the case, makes sense! Same applies to all the other db stuff.